python/whylogs/core/preprocessing.py

```diff
@@ -282,10 +304,12 @@ def apply(data: Any) -> "PreprocessedColumn":
                 floats = np.asarray(float_list, dtype=float)
 
                 result.numpy = NumpyView(ints=ints, floats=floats)
-                result.list = ListView(strings=string_list, objs=obj_list)
+                result.list = ListView(strings=string_list, tensors=tensor_list, objs=obj_list)
                 return result
             else:
-                result.list = ListView(ints=int_list, floats=float_list, strings=string_list, objs=obj_list)
```

Comment:
`assert tensor_list == []` ?

Reply:
I think this particular one was just a miss.

Reply:
It was just a suggestion to check the invariant here.